src/rust/ensogl/lib/text/src/component/area.rs

```diff
@@ -673,6 +697,7 @@ impl Area {
 #[derive(Clone,CloneRef,Debug)]
 pub struct AreaData {
     scene          : Scene,
+    camera         : Rc<CloneRefCell<display::camera::Camera2d>>,
```

Comment:
Ok, I understand why this field is here, but I trully displike this solution. Could you add fixme here that it should be imporved once we have better object management on the scene, please?